### DISCUSSION TITLE
[1.19.2]: Barbecue Stick ingredients should use a tag

### DISCUSSION THREAD

**Discussion by bageldotjpg** [vectorwing/FarmersDelight#751] [Ideas]
Like Cabbage Rolls, Barbecue Sticks should use a `forge:cooked_meat` tag or a `farmersdelight:barbecue_stick_ingredients` tag instead of the list of possible meats in the recipes, to make it easier to add support for.

**Comment by thelegitdolt** [comment]
Same with Bone Broth, I think

**Comment by vectorwing** [comment]
I was a bit afraid of adding yet another tag for `specific_recipe_ingredients`, as it seemed like an excess of tags for small features. But given some of them are very broad, like Barbecue Stick, this could have value.

I'll see about organizing those recipes better later, as I have some recipe rebalances planned.

**Comment by thelegitdolt** [comment]
you could make an Ingredient folder inside the item folder for organization

**Comment by vectorwing** [comment]
Moved this over to Discussions, as I would need to refine this further before deciding on something.
